versions.json

```diff
@@ -100,88 +100,88 @@
-                "sha256": "aa28fa2227768dd31ebbf9cd48b265a0c8810fae03e02c6079c0fa71bbea7319"
+                "version": "12.5",
+                "sha256": "5c40cc096706045b067ec5897f039403014aa7a39b970905698466a2d029b972"
             }   
         },
         "almalinux8.7": {
             "driver": {
-                "version": "12-4",
+                "version": "12.5",
                 "distribution": "rhel8"
             },
             "samples": {
-                "version": "12.4",
-                "sha256": "aa28fa2227768dd31ebbf9cd48b265a0c8810fae03e02c6079c0fa71bbea7319"
+                "version": "12.5",
+                "sha256": "5c40cc096706045b067ec5897f039403014aa7a39b970905698466a2d029b972"
             }   
         }
     },
     "gdrcopy": {
         "ubuntu20.04": {
-            "version": "2.3",
-            "sha256": "b85d15901889aa42de6c4a9233792af40dd94543e82abe0439e544c87fd79475",
+            "version": "2.4.1",
+            "sha256": "faa7e816e9bad3301e53d6721457f7ef5ab42b7aa3b01ffda51f8e5620bb20ed",
             "distribution": "Ubuntu20_04"
         },
         "ubuntu22.04": {
-            "version": "2.3",
-            "sha256": "b85d15901889aa42de6c4a9233792af40dd94543e82abe0439e544c87fd79475",
+            "version": "2.4.1",
+            "sha256": "faa7e816e9bad3301e53d6721457f7ef5ab42b7aa3b01ffda51f8e5620bb20ed",
             "distribution": "Ubuntu22_04"
         },
         "almalinux8.7": {
-            "version": "2.3",
-            "sha256": "b85d15901889aa42de6c4a9233792af40dd94543e82abe0439e544c87fd79475",
+            "version": "2.4.1",
+            "sha256": "faa7e816e9bad3301e53d6721457f7ef5ab42b7aa3b01ffda51f8e5620bb20ed",
             "distribution": "el8"
         }
     },
     "nccl": {
         "common": {
-            "version": "2.21.5-1",
+            "version": "2.22.3",
             "rdmasharpplugins": {
-                "commit": "575c1e0"
+                "commit": "v2.6.x"
             }
         }
     },
     "dcgm": {
         "common": {
-            "version": "3.3.3"
+            "version": "3.3.7"
         }
     },
     "intel_one_mkl": {
         "common": {
-            "version": "2024.0.0.49673",
-            "sha256": "2a3be7d01d75ba8cc3059f9a32ae72e5bfc93e68e72e94e79d7fa6ea2f7814de",
-            "url": "https://registrationcenter-download.intel.com/akdlm/IRC_NAS/86d6a4c1-c998-4c6b-9fff-ca004e9f7455/l_onemkl_p_2024.0.0.49673_offline.sh"
+            "version": "2024.2.0.664",
+            "sha256": "f1f46f5352c197a9840e08fc191a879dad79ebf742fe782e386ba8006f262f7a",
+            "url": "https://registrationcenter-download.intel.com/akdlm/IRC_NAS/cdff21a5-6ac7-4b41-a7ec-351b5f9ce8fd/l_onemkl_p_2024.2.0.664_offline.sh"
         }
     },
     "waagent": {
         "ubuntu20.04": {
-            "version": "2.9.0.4",
-            "sha256": "040969f507f73f3a2c95d5b0568225ad68f7f91bfec99bd92154c3fa9e28034b"
+            "version": "2.11.1.4",
```

Comment:
Possible to use a common version for waagent?

Reply:
possible, but highlighting for the moment that alma DOESN'T use the version file. I'd like to see if there's a common install strategy we want to use for WAAGENT, but didn't want to press to do it in this PR